acs/acs.go

```diff
@@ -0,0 +1,37 @@
+package acs
+
+type ACS interface {
```

Comment:
What is your intent to implement `ACS` as interface?

Reply:
For scalability, maybe we can re-use it in tc or other dependency.